Add management-api documentation (#228)
wso2/micro-integrator: doc/management-api.md

> # WSO2 Micro Integrator Management Api

Reviewer: ```suggestion
# WSO2 Micro Integrator REST API
```

> This specifies a RESTful API for WSO2 Micro Integrator Management

Reviewer: ```suggestion
This describes the API resources that make up the Micro Integrator RESTful API.
```

> Please see [full swagger definition](https://raw.githubusercontent.com/wso2/micro-integrator/master/doc/management-api.yaml) of the API which is written using [swagger 2.0](http://swagger.io/) specification.

Reviewer: ```suggestion
Please see [full swagger definition](management-api.yaml) of the API which is written using [swagger 2.0](http://swagger.io/) specification.
```

> ## Version: 1.0.0-RC1

Reviewer: ```suggestion
## Version: 1.0.0
```

**License:** [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)

### /applications

#### GET
##### Summary:

Retrieve Applications

> 

Reviewer: ```suggestion
```
